src/store/getters.js

```diff
@@ -32,7 +32,7 @@ export default {
 		for (const key in addons) {
 			switch (addons[key].state) {
 				case "enabled":
-					addons[key] = { enabled: true, disabledReason: "" };
+					addons[key] = { enabled: true, behaviour: addons[key].behaviour, disabledReason: "" };
```

Comment:
this means that only the components will have a behaviour prop on the addon, right?

Reply:
yeah only the components will use this behaviour. not sure if I should add and `if` case for this.